pymodbus/server/async_io.py

```diff
@@ -203,14 +207,12 @@ def execute(self, request, *addr):
                 traceback.format_exc(),
             )
             response = request.doException(merror.SlaveFailure)
-        # no response when broadcasting
-        if not broadcast:
-            response.transaction_id = request.transaction_id
-            response.slave_id = request.slave_id
-            skip_encoding = False
-            if self.server.response_manipulator:
-                response, skip_encoding = self.server.response_manipulator(response)
-            self.send(response, *addr, skip_encoding=skip_encoding)
+        response.transaction_id = request.transaction_id
```

Comment:
This changes functionality, I am pretty sure that when broadcasting the devices do not respond (Have to check the spec on that).

Reply:
Note the [return](https://github.com/pymodbus-dev/pymodbus/blob/41b2d3a601c1a2da081649b16a3510c6dd57ae53/pymodbus/server/async_io.py#L189) after the loop.

Reply:
? you removed the if which secured broadcast is not handled.